src/plugins/data/public/search/errors/painless_error.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { get } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { ApplicationStart } from 'kibana/public';
+import { KbnError } from '../../../../kibana_utils/common';
+import { IEsSearchRequest } from '..';
+
+export class PainlessError extends KbnError {
+  constructor(err: Error, request: IEsSearchRequest) {
+    const rootCause = get(err, 'body.attributes.error.root_cause');
+    const [{ script }] = rootCause;
+
+    super(
+      i18n.translate('data.painlessError.painlessScriptedFieldErrorMessage', {
+        defaultMessage: "Error with Painless scripted field '{script}'.",
+        values: { script },
+      })
+    );
+  }
+
+  public getErrorMessage(application: ApplicationStart) {
+    function onClick() {
+      application.navigateToApp('management', {
+        path: `/kibana/indexPatterns`,
+      });
+    }
+
+    return (
+      <>
+        {this.message}
+        <EuiSpacer size="s" />
+        <div className="eui-textRight">
```

Comment:
Perhaps `EuiText` or `EuiTextAlign` [(doc link)](https://eui.elastic.co/#/display/text) might be better here and we can then lose the class